createMeanyButelkaObjects.cls

```diff
@@ -0,0 +1,44 @@
+public  class createMeanyButelkaObjects {
+Integer howManyBottleToCreate ;
+public static Integer bottleNumber=0;
+
+public static void createMoreBottleObjects (Integer howManyBottleToCreate){
+
+        List<butelka> bottleObjectsList = new List <butelka>();
+        
+            for(Integer i=0; i< howManyBottleToCreate; i++){
+                Integer newBottleNumber = bottleNumber++;
```

Comment:
generalnie to jako zajawke powiem Ci, że bottleNumber jaki nadajesz i w jaki sposób go pewnie wykorzystujesz, mozesz zastąpić tym, że będziesz ściągał elemnty z listy, bo tutaj też nadajesz i dodajesz je po kolei, ale to tylko adnotacja, generalnie zrobione jest sensownie